### PR TITLE
docs(success-stories): Add Biden for President quote

### PR DESCRIPTION
**Pull request by prophen**
I added the quote from Matt Hodges, Engineering Director, Biden For President to the testimonials section in `/success-stories`

### CODE REVIEW

**Comment by prophen** [comment]
@brian-armory please review, thank you

**Review by aimeeu — CHANGES_REQUESTED** [review]
@prophen Nikema - thanks for the update. Couple of concerns - since there's no logo, that space appears as a broken image.  Also, there's nothing in the "Click here to learn more" section.  It looks like these success stories involved interviewing the user? Not really sure how to proceed with this.

https://s.armory.io/WnuBoq05

**Comment by prophen** [comment]
> @prophen Nikema - thanks for the update. Couple of concerns - since there's no logo, that space appears as a broken image. Also, there's nothing in the "Click here to learn more" section. It looks like these success stories involved interviewing the user? Not really sure how to proceed with this.
> 
> https://s.armory.io/WnuBoq05

Thanks, @aimeeu. I addressed your concerns with the latest commit. The quote is now in the case studies section along with an image so it doesn't break the design 
image

File: _data/success-stories.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@ cases:
     logo: cases-logos/waze.jpg
     text: Thanks to Spinnaker, developers can focus on developing business logic rather than becoming experts on each cloud platform. Teams can track the lifecycle of a deployment using several notification mechanisms including email, Slack, and SMS, allowing them to coordinate handoffs between developer and QA teams.
 
+  - name: Biden for President
+    link: https://spinnaker.io/guides/
```
Comment (aimeeu) [inline _data/success-stories.yml]: Wondering why this link is to the Spinnaker docs? In other Case Studies, the "Learn" link goes to a whitepaper or external site with an actual case study. Is it possible to link to wherever that quote appeared?  twitter, news site, ??

**Review by aimeeu — CHANGES_REQUESTED** [review]

**Comment by prophen** [comment]
image I created a new section for the quote so that it isn't out of place with the testimonials or case studies. What do you think, @aimeeu ?

**Review by aimeeu — APPROVED** [review]

**Pull request merged**